### DISCUSSION TITLE
Omeka-s en Raspberry

### DISCUSSION THREAD

**Discussion by bizkeliza** [community/community#76395] [Packages]
### Select Topic Area

Question

### Body

Hola, ¿Es posible instalar omeka-s en raspberry? 
Hello, Is it possible to install omeka-s on raspberry?
Thanks!

**Comment by Akash1134** [comment]
Hi @bizkeliza , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
